app/src/main/java/com/uncookthebook/app/models/Website.java

```diff
@@ -0,0 +1,25 @@
+package com.uncookthebook.app.models;
+
+import com.google.gson.annotations.SerializedName;
+
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.ToString;
+
+/**
+ * Models a Website.
+ */
+@Getter
+@ToString
+public class Website implements Model {
+    @SerializedName("name")
+    private final String name;
+
```

Comment:
What about his rating?

Reply:
To be done. I need to do redundancy analysis on the DB to know if it is better to have the rating or not